### ISSUE TITLE
IsInRole does not work

### ISSUE THREAD

**Issue by kwerty** [enkodellc/blazorboilerplate#186]
I'm mucking around with the NavMenu.razor template (down the bottom where the login state is checked).

'This returns false, even though the user is in the Administrator role
` user.IsInRole("Administrator")`

If we change the code to this, it returns true:
` user.Claims.Any(x => x.Type == "role" && x.Value == "Administrator")`

As I understand it, the `ClaimsPrincipal.IsInRole()` function is looking for a claim with type `"http://schemas.microsoft.com/ws/2008/06/identity/claims/role"` and value `Administrator`, but it's not finding it, and thus returning false.

It seems roles are being added to `ClaimsPrincipal.Claims` with type `role`, rather than `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name`. I can't figure out where this is happening.

This stackoverflow thread adequately explains the problem
https://stackoverflow.com/questions/39884810/why-net-core-has-its-own-claim-types

The tl;dr is that user.IsInRole doesn't work, and it's been driving me mad for the past few hours.

Appreciate your guys hard work on the boilerplate.

**Comment by kwerty** [comment]
Found the fix here - https://leastprivilege.com/2016/08/21/why-does-my-authorize-attribute-not-work/

Line 96, change
`identity = new ClaimsIdentity(claims, "Server authentication");`
to
`identity = new ClaimsIdentity(claims, "Server authentication", "name", "role");`

**Comment by enkodellc** [comment]
@kwerty I am pretty confident that it was working at some point as I try my best to test everything but as the project grows and new versions come out it is quite a challenge to maintain. I will update / test with your suggested fix. Also take a look at this PR as I think it will be in our next release: https://github.com/enkodellc/blazorboilerplate/pull/184

**Comment by enkodellc** [comment]
Seems to fix. I
